src/EnergyPlus/Autosizing/HeatingWaterDesCoilLoadUsedForUASizing.cc

```diff
@@ -130,6 +130,7 @@ Real64 HeatingWaterDesCoilLoadUsedForUASizer::size(EnergyPlusData &state, Real64
             }
         }
     } else if (this->curSysNum > 0) {
+        this->dataFracOfAutosizedHeatingCapacity = 1.0;
```

Comment:
This line should not be needed since dataFracOfAutosizedHeatingCapacity is set in BaseSizerWithFanHeatInputs::initializeWithinEP as

    this->dataFracOfAutosizedHeatingCapacity = state.dataSize->DataFracOfAutosizedHeatingCapacity;

and state.dataSize->DataFracOfAutosizedHeatingCapacity is initialized to 1 in the DataSizing struct

    Real64 DataFracOfAutosizedHeatingCapacity = 1.0;

If you have a test file this is easily verified by putting a breakpoint at line 133 and checking what the value is before this line executes. If not test file, any example file with a HW coil that is autosized will do.

Reply:
I see this same thing at line 217 of HeatingCapacitySizing.cc. That line could be removed as well if this turns out the way I think it should.